### ISSUE TITLE
G6 mg/dl app, no value in AAPS

### ISSUE THREAD

**Issue by mcdviper** [dexcomapp/dexcomapp#6]
I don't get any values in AAPS with the G6 mg/dl app from feb 7.
As I'm pretty new to this, are there specific settings needed?
I don't share.

Thanks

**Comment by philipgo2** [comment]
You need the developer version of AAPS, only this version allows to select G6 as source in AAPS.

**Comment by mpessnegger** [comment]
I have the same issue. Just changed from Libre/MiaoMiao to Dexcom G6. i selected „DexcomG6 App (gepatched) in my AAPS 2.3. but i dont get any values transmitted directly from app to app. As a current dummy solution, i send the G6 Values to Nightscout and collect those values then on AAPS. but if i am in flightmode or do not have any internet connection, i cannot use the system.

using AAPS 2.3 Build: c0cbaafe0-2019.05.03-09:07 and Samsung S4-Mini Android version 8.0.1 with patched G6 app mgdl region3 (but for AAPS 2.4, since the patched G6 app for AAPS 2.3 was not able to start the sensor). could that patched G6 app difference (2.3 / 2.4) be the reason, everything works fine except the app to app value transfer?

Thanks for any help
Max

**Comment by billymild** [comment]
Did this get fixed?

**Comment by mpessnegger** [comment]
working with AAPS 2.5.1 properly

**Comment by billymild** [comment]
Did you have to enable Share for it to work? I am not getting glucose numbers from the patched app into AAPS.

**Comment by mpessnegger** [comment]
yes , i have share enabled for getting the values directly from dexcom app into nightscout. but i don’t know if this has to be done to get it trasnfered between apps. with same settings it did not work with AAPS 2.3 but finally since update to 2.5

maybe that helps:
https://docs.google.com/forms/d/e/1FAIpQLScD76G0Y-BlL4tZljaFkjlwuqhT83QlFM5v6ZEfO7gCU98iJQ/viewform

there you can compile your own dexcom app. but i dont know, if that will tranfer the values between dexcom and AAPS, since i use the app from github. i wouls appreciate, if you can feedback whether or not that option solves your issue
